### DISCUSSION TITLE
Adding the native size attribute to the Input elements

### DISCUSSION THREAD

**Discussion by ChrisFrankoPhD** [mantinedev/mantine#6269] [Q&A] [answered]
Is there a way to pass a custom "size" attribute to the native HTML input element in Mantine's inputs? Using the styles API for example we can add custom classes and inline styling to the input via `className={classNames({ input: 'bg-slate-600' })}`. This is great, but does not work for non-style native attributes like the HTML size attribute. Is there a way to pass that from the parent component? 

For context I want the width of a given text input to be the character length of the current value. The width style does not track this well, but using a non-mantine component its easy to set `size=value.length`

**Comment by rtivital** [comment, marked as answer]
It is not currently possible. I'll add this feature in one of the next releases.

**Reply by ChrisFrankoPhD** [reply]
thank you for the reply, I appreciate it, would be great!
